rita/src/rita_client/exit_manager/mod.rs

```diff
@@ -259,15 +262,14 @@ impl Handler<Tick> for ExitManager {
                     ));
                 }
                 ExitState::Registering { .. }
-                | ExitState::Pending { .. }
                 | ExitState::GotInfo {
                     auto_register: true,
                     ..
                 } => {
-                    Arbiter::handle().spawn(exit_setup_request(k.clone()).then(move |res| {
+                    Arbiter::handle().spawn(exit_setup_request(k.clone(), None).then(move |res| {
```

Comment:
what's the arbiter doing here?

Reply:
why can't we use a more normal future?

Reply:
Normally you will have to spawn your future into a tokio core, which you start yourself, but since we have a cute running anyways from actix, we may as well use it because it's inefficient to run 2 event loops when you can do one